iceoryx_hoofs/source/log/logger.cpp

```diff
@@ -123,21 +136,25 @@ void Logger::Print(const LogEntry entry) const
     auto sec = std::chrono::duration_cast<std::chrono::seconds>(entry.time);
     std::time_t time = sec.count();
 
-    auto timeInfo = std::localtime(&time);
+    // TODO: std::localtime is thread-unsafe, may be remove
```

Comment:
Can you elaborate this comment in more detail. How is `std::localtime` thread-unsafe? How would affect it the following code?